docs/application-connector/docs/03-07-api-registration.md

```diff
@@ -0,0 +1,21 @@
+---
+title: API registration in Application Registry
+type: Details
+---
+
+### Supported formats
+
+The Application Registry supports API Specification in the following formats:
+- OpenAPI 2.0
+- OData 2.0
+
+### Registration methods
+
+Application Registry supports two ways of passing API spec:
+- in JSON format
+- by `SpecificationUrl`
+
+>**NOTE:** Specification passed directly as a JSON has a higher priority than `SpecificationUrl`, which in such case will be ignored.
```

Comment:
```suggestion
>**NOTE:** Specification passed directly as a JSON has a higher priority than `SpecificationUrl`.  If you use these two methods at once, `SpecificationUrl` is ignored.
```

Reply:
I'd move this note at the end of the document.

Reply:
Done!